### PR TITLE
Rebuild py38 migration for hangar-feedstock

### PR DESCRIPTION
**Pull request by rlizzo**
Addresses bug in https://github.com/conda-forge/hangar-feedstock.

Submitting PR as advised by @CJ-Wright in https://github.com/regro/cf-scripts/issues/946

### CODE REVIEW

File: node_attrs/hangar.json [hunk 1]
```diff
@@ -34,25 +34,6 @@
     "version"
    ]
   },
-  {
-   "PR":{
-    "__lazy_json__":"pr_json/124039c9-3f22-4126-b5d1-6cfc674baaa6.json"
-   },
-   "data":{
-    "bot_rerun":false,
```
Comment (CJ-Wright) [inline node_attrs/hangar.json:42]: I'm very sorry can you just change this line to `"bot_rerun":1.0,`? That should trigger a new PR without causing us to loose the information in that pr (for debugging)

**Comment by rlizzo** [comment]
No problem, let me know if there's anything else I can do to assist :)

**Comment by CJ-Wright** [comment]
Let's see how this goes!

**Comment by CJ-Wright** [comment]
sorry for the delay, we try not to update the metadata when the bot is running in case it can't merge properly

**Comment by rlizzo** [comment]
No worries; It's an impressive system you guy have built - though I can only imagine how much effort goes into upkeep/administration. It's more than worth a few hours wait for one or two "problem-children" if it reduces the chances that something might go wrong in a way that affects the larger ecosystem/devs/userbase.

I'll follow up in the Issue thread once we know if this worked or not. Thanks again for the help.

**Pull request merged**